components/GlobalStyle.js

```diff
@@ -17,26 +34,63 @@ html {
 }
 
 
-/* noto-sans-regular - latin */
+
+
+/* montserrat-alternates-regular - latin */
 @font-face {
-  font-family: 'Noto Sans';
+  font-family: 'Montserrat Alternates';
   font-style: normal;
   font-weight: 400;
   src: local(''),
-       url('/fonts/noto-sans-v27-latin-regular.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
-       url('/fonts/noto-sans-v27-latin-regular.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+       url('/fonts/montserrat-alternates-v17-latin-regular.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
+       url('/fonts/montserrat-alternates-v17-latin-regular.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
 }
 
 
+/* dm-sans-regular - latin */
+@font-face {
+  font-family: 'DM Sans';
+  font-style: normal;
+  font-weight: 400;
+  src: local(''),
+       url('/fonts/dm-sans-v11-latin-regular.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
+       url('/fonts/dm-sans-v11-latin-regular.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+
+ 
+  h1 {
+    font-size: 2rem;
+    color: var(--primary);
+    text-align: center;
+    margin: 1rem 0;
+    font-family: "Montserrat Alternates", "Open Sans", Helvetica, Arial, sans-serif;
+  }   
+
+  h2 {
+    font-size: 1.5rem;
+    color: var(--primary);
+    text-align: center;
+    margin: 1rem 0;
+    font-family: "Montserrat Alternates", "Open Sans", Helvetica, Arial, sans-serif;
+  }
+
+  h3{
+    font-size: 1.25rem;
+    color: var(--text-dark);
+    text-align: center;
+    margin: 1rem 0;
+    font-family: "Montserrat Alternates", "Open Sans", Helvetica, Arial, sans-serif;
+  }
   
-      * {
-          box-sizing: border-box;
-      }
```

Comment:
Did you mean to delete the border-box or did you removed it by accident? Just curious. Border box usually simplifies the sizing of elements.

Reply:
Hi Ernst, thanks for your feedback! I accidentally had the border-box rule twice so I deleted it once ;)